schemas/tenant.py

```diff
@@ -1,13 +1,30 @@
 from ma import ma
 from models.tenant import TenantModel
-from marshmallow import fields
+from marshmallow import fields, validate
 from utils.time import time_format
 
 
 class TenantSchema(ma.SQLAlchemyAutoSchema):
-  class Meta:
-    model = TenantModel
+    class Meta:
+        model = TenantModel
 
-  addedOn = fields.DateTime(time_format)
-  created_at = fields.DateTime(time_format)
-  updated_at = fields.DateTime(time_format)
+    created_at = fields.DateTime(time_format)
+    updated_at = fields.DateTime(time_format)
+
+    firstName = fields.Str(
+        required=True,
+        validate=validate.Length(min=1, max=100),
```

Comment:
I'm not familiar with this method. It seems assigning it to validate has some automatic message handling happening? Is that correct?

Reply:
Yes, I just used one of the built-in validators for this:
https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html#api-validators

Looking at the source code, each one has built-in message(s) if you don't provide the optional `error` argument:
https://marshmallow.readthedocs.io/en/stable/_modules/marshmallow/validate.html#Length

Reply:
good to know! Thank you.